frappe/website/web_template/discussions/discussions.html

```diff
@@ -1,3 +1,5 @@
 {% set doctype = "Web Page" %}
+{% set empty_state_title = "No " + title %}
+{% set empty_state_subtitle = "There are no " + title.lower() + " for this " + doctype.lower() + ", why don't you start one!" %}
```

Comment:
User facing text should be translated and should not be split into parts.

https://frappeframework.com/docs/user/en/translations#3-blocks